org.virtualbox.VirtualBox.yaml

```
@@ -0,0 +1,207 @@
+app-id: org.virtualbox.VirtualBox
+runtime: org.kde.Platform
+runtime-version: "6.9"
+sdk: org.kde.Sdk
+command: VirtualBox
+rename-icon: virtualbox
+rename-desktop-file: virtualbox.desktop
+finish-args:
+  # Default path that VirtualBox places files at
+  - "--persist=~/VirtualBox VMs"
```

```suggestion
  - "--persist=VirtualBox VMs"
```

it's always relative to $HOME

---

oh yeah oops i did this change without realising i left the ~ x.x

---

https://github.com/flathub/flathub/pull/6985/commits/8b4b001dbdd0eecdc2217ee643a720df1a0524bd